weights_thickness.m

```diff
@@ -58,6 +58,6 @@
 vol_ballast = wt_ballast./(c.rho_b.*g);
 
 %Height of Ballast
-height_ballast = vol_ballast./(g.*c.rho_b.*pi.*R1^2);
+height_ballast = vol_ballast./(g.*c.rho_b.*pi.*R1.^2);
```

Comment:
g and rho should be deleted

Reply:
I think you were looking at an older version